### PR TITLE
Update the repository readme

### PR DESCRIPTION
**Pull request by akochari**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description

Updated the description of the service and contributing info.

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [ ] New feature
- [x] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [ ] I have added a reviewer for this pull request
- [ ] I have added myself as an author for this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,67 @@
 
-# Studio
+# SciLifeLab Serve
```
Comment (sandstromviktor) [inline README.md:2]: Perhaps we can add github action badges aswell?
example

Comment (sandstromviktor) [reply README.md:2]: We should change the workflow name to serve instead of studio

**Review by sandstromviktor — APPROVED** [review]
Nice improvement to our repo!

File: README.md [hunk 2]
```diff
@@ -1,62 +1,67 @@
...
 
-## Important
+SciLifeLab Serve ([https://serve.scilifelab.se](https://serve.scilifelab.se)) is a platform offering machine learning model serving, app hosting (Shiny, Streamlit, Dash, etc.), web-based integrated development environments, and other tools to life science researchers affiliated with a Swedish research institute. It is developed and operated by the [SciLifeLab Data Centre](https://github.com/ScilifelabDataCentre), part of [SciLifeLab](https://scilifelab.se/). See [this page for information about funders and mandate](https://serve.scilifelab.se/about/).
 
-This is a private repository and some django app dependencies (studio-user, studio-control etc) are also private. To build studio locally you need to have ssh authentication enabled for Github.
-I recommend you add it to your ssh-agent. App dependencies will be installed via pip git+ssh.
+This repository contains the code behind SciLifeLab Serve. It is  based on the open-source platform [Stackn](https://github.com/scaleoutsystems/stackn).
```
[the remark this answers is not in the document]
Comment (churnikov) [reply README.md:6]: We could write like this if you will:
This repo contains source code for Scilifelab serve.

**Pull request merged**